x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -105,11 +106,20 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
   const [isEnabled, setIsEnabled] = useState<boolean>(alert.enabled);
   const [isMuted, setIsMuted] = useState<boolean>(alert.muteAll);
   const [editFlyoutVisible, setEditFlyoutVisibility] = useState<boolean>(false);
+  const [dissmissAlertErrors, setDissmissAlertErrors] = useState<boolean>(false);
 
   const setAlert = async () => {
     history.push(routeToAlertDetails.replace(`:alertId`, alert.id));
   };
 
+  const getAlertStatusErrorReasonText = () => {
+    if (alert.executionStatus.error && alert.executionStatus.error.reason !== 'unknown') {
+      return `An error occurred when ${alert.executionStatus.error.reason} the alert.`;
+    } else {
+      return 'An error occurred for unknown reasons.';
+    }
```

Comment:
This should be a switch case and call i18n.translate(...) per reason so `alert.executionStatus.error.reason` can be translated.